index.js

```diff
@@ -48,6 +48,7 @@ export default class TextInputMask extends Component {
   render() {
     return (<TextInput
       {...this.props}
+      collapsable={false}
```

Comment:
I think I added that before I figured out why the actual crashes happened, I don’t think it makes a difference at all since TextInput is a native component and can’t be collapsed. I’ll remove it.

Reply:
For reference here’s the doc for the prop on View:
https://facebook.github.io/react-native/docs/view#collapsable

Originally I thought the view was being collapsed and thus wasn’t found in the native hierarchy.